ambry-store/src/main/java/com/github/ambry/store/BlobStoreCompactor.java

```diff
@@ -551,17 +552,23 @@ private boolean isIndexSegmentUnderCopy(Offset indexSegmentStartOffset) {
   private boolean copyDataByLogSegment(LogSegment logSegmentToCopy, FileSpan duplicateSearchSpan)
       throws IOException, StoreException {
     logger.info("Copying data from {}", logSegmentToCopy);
+    long logSegmentStartTime = SystemTime.getInstance().milliseconds();
```

Comment:
can we use `time.milliseconds()` ?

Reply:
we should probably use System.currentMillis(); this is for metrics.

Reply:
The `time.miliseconds()` actually calls `System.currentMillis()` (`MockTime` isn't used in production code), so I think either is fine.

Reply:
changed to time.